SensusService/Probes/PollingProbe.cs

```diff
@@ -208,7 +237,32 @@ protected override void InternalStart()
 
                     }, GetType().FullName + " Poll", TimeSpan.FromMinutes(_pollingTimeoutMinutes), userNotificationMessage);
 
+#if __IOS__
+                if (_significantChangePoll)
+                {
+                    _locationManager.RequestAlwaysAuthorization();
+                    _locationManager.DistanceFilter = 5.0;
+                    _locationManager.PausesLocationUpdatesAutomatically = true;
```

Comment:
Sorry, Wes's stuff. I didn't realize some o f his changes had been mixed in with mine.